README.md

```diff
@@ -16,6 +16,14 @@ Security audit for [v0.4 version of the contracts](./contracts/v0.4/) is availab
 - Name: ChainLink Token
 - Symbol: LINK
 
+## ABI
```

Comment:
Hey @PatrickAlphaC, the ABIs for respective versions are already available here:

- https://github.com/smartcontractkit/LinkToken/tree/master/contracts/v0.4
- https://github.com/smartcontractkit/LinkToken/tree/master/contracts/v0.6

If users miss it, maybe we could just point to it more explicitly instead of duplicating the ABI here?

Reply:
Yup, that makes way more sense.